### DISCUSSION TITLE
Add Playtime top Placeholder

### DISCUSSION THREAD

**Discussion by YusakiDev** [Zrips/CMI#4830] [Enhancement]
It should have playtimetop like baltop, Players like to have their name display on their top leaderboards

**Comment by mrfloris** [comment]
already a feature, but since this doens't have a response.

in console type ` cmi placeholders ` and review what placeholders are available, to find all the 'top' ones. including playtime.

**Comment by mrfloris** [comment]
OH, i understand you now
/baltop has a placeholder that can be used in a leaderboard to say : your balance ranks you #5  ... 
/playtimetop does not have such a placeholder. we can make top placehodlers, but we can't say: your playtime ranking is #12

**Comment by CloudeLecaw** [comment]
the requested placeholder was added with CMI 9.8.2.4 as `%cmi_user_playtimetop%`
